source/uwp/AdaptiveCardTestApp/Expected/SourceCards/Tests.ToogleVisibility.AllElements.beea57a.json

```diff
@@ -0,0 +1,276 @@
+{
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "type": "AdaptiveCard",
+    "version": "1.0",
+    "body": [
+		{
+			"type": "Image",
+			"url": "http://adaptivecards.io/content/cats/1.png",
+			"id": "id1",
+			"selectAction": {
+                "type": "Action.ToggleVisibility",
+                "targetElements": [
+                    "id2"
+                ]
+            }
```

Comment:
>             [](start = 0, length = 12)

This result file still has spaces in it. You should re-run the test app as i think that will count as "source changed" for these results.